src/main/menu_main.cpp

```diff
@@ -1661,16 +1691,16 @@ void mainMenuDraw()
     if(MenuMode == MENU_MAIN)
     {
         int i = 0;
-        SuperPrint(g_gameInfo.disableTwoPlayer ? g_mainMenu.mainStartGame : g_mainMenu.main1PlayerGame, 3, 300, 350+30*(i++));
+        SuperPrint(g_gameInfo.disableTwoPlayer ? g_mainMenu.mainStartGame : g_mainMenu.main1PlayerGame, 3, MenuX, MenuY+30*(i++));
         if(!g_gameInfo.disableTwoPlayer)
-            SuperPrint(g_mainMenu.mainMultiplayerGame, 3, 300, 350+30*(i++));
+            SuperPrint(g_mainMenu.mainMultiplayerGame, 3, MenuX, MenuY+30*(i++));
         if(!g_gameInfo.disableBattleMode)
-            SuperPrint(g_mainMenu.mainBattleGame, 3, 300, 350+30*(i++));
+            SuperPrint(g_mainMenu.mainBattleGame, 3, MenuX, MenuY+30*(i++));
         if(g_config.enable_editor)
-            SuperPrint(g_mainMenu.mainEditor, 3, 300, 350+30*(i++));
-        SuperPrint(g_mainMenu.mainOptions, 3, 300, 350+30*(i++));
-        SuperPrint(g_mainMenu.mainExit, 3, 300, 350+30*(i++));
-        XRender::renderTexture(300 - 20, 350 + (MenuCursor * 30), 16, 16, GFX.MCursor[0], 0, 0);
+            SuperPrint(g_mainMenu.mainEditor, 3, MenuX, MenuY+30*(i++));
+        SuperPrint(g_mainMenu.mainOptions, 3, MenuX, MenuY+30*(i++));
+        SuperPrint(g_mainMenu.mainExit, 3, MenuX, MenuY+30*(i++));
+        XRender::renderTexture(MenuX - 20, MenuY + (MenuCursor * 30), 16, 16, GFX.MCursor[0], 0, 0);
```

Comment:
At any arithmetical expression, it's better to always add space between operands and operators. Also, when it's a multiplication and adding `x + y * z`, better to add parens just in case `x + (y * z)`. I do know some faulty compilers that result in the wrong result (some very old GCC, but I don't remind the exact place where I caught this mess...).